compiler/rustc_parse/src/validate_attr.rs

```diff
@@ -267,6 +267,36 @@ pub fn check_builtin_meta_item(
     deny_unsafety: bool,
 ) {
     if !is_attr_template_compatible(&template, &meta.kind) {
+        // attrs with new parsers are locally validated so excluded here
```

Comment:
This move is in a separate commit.
It is needed because https://github.com/rust-lang/rust/blob/1ce9c977ffcff7c3b12bfe5629a682da0e74a7a1/compiler/rustc_parse/src/validate_attr.rs#L359 also calls emit_malformed_attribute and is not using the new parser (yet?), so should still emit the lint even for parsed attributes